Dockerfile

```
@@ -14,17 +14,12 @@ USER root
 #
 # We pin nodejs 16.x instead of accepting Playwright's default for consistency with
 # our other build environments.
-#
-# dotnet-sdk-6.0 is required to build mock-adb
 RUN apt-get update && \
```

On modern Debian-based systems:

```suggestion
RUN apt update && \
```